splice_machine/src/main/java/com/splicemachine/derby/impl/sql/catalog/upgrade/SpliceCatalogUpgradeScripts.java

```diff
@@ -76,6 +76,7 @@ else if(v1==v2)
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1933), new UpgradeScriptToUpdateViewForSYSCONGLOMERATEINSCHEMAS(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1938), new UpgradeScriptForTriggerWhenClause(sdd,tc));
         scripts.put(new Splice_DD_Version(sdd,3,1,0, 1941), new UpgradeScriptForTableColumnViewInSYSIBM(sdd,tc));
+        scripts.put(new Splice_DD_Version(sdd,3,1,0, 1949), new UpgradeScriptBase(sdd,tc));
```

Comment:
I just have a minor comment to change 1949 to 1948 in anticipation that this PR will be merged to the 1948 build. So if the system is already running 1948, this script won't be triggered again when upgrading to 1949 (a script will be trigged only if the system is running a version lower than the build number specified here. If the running version is same as the build number, SpliceDataDictionary.needToUpgrade() ensures that no upgrade script will be triggered).

Reply:
Done.